### DISCUSSION TITLE
Working Model Review

### DISCUSSION THREAD

**Discussion by ma3u** [catenax-eV/catenax-ev.github.io#95] [General]
The Catena-X association need your input for the Working Model:
1. [Committee](https://catenax-ev.github.io/docs/next/working-model/organizational-structure/catena-x/committee)
2. [Expert Groups](https://catenax-ev.github.io/docs/next/working-model/organizational-structure/catena-x/expert-group)
3. [Release Management](https://catenax-ev.github.io/docs/next/working-model/release-management)
4. [Process Structure](https://catenax-ev.github.io/docs/next/working-model/process-structure) - Detailed description need to be cleaned up
5. [Test management](https://catenax-ev.github.io/docs/next/working-model/testmanagement)

Work on the branch e.g. [adapt-structure](https://github.com/catenax-eV/cx-working-model/tree/doc/adapt-structure/docs). Our committee can contribute directly in respected Github organization **catenax-eV** and the private repo [cx-working-model](https://github.com/catenax-eV/cx-working-model). Use this private Repo to contribute directly with a Pull-Request with Reviewer @Grand-Thibault and @stephanbcbauer. 

use Markdown Linter in Visual Studio Code (App or Online): markdownlint

**Comment by ma3u** [comment]
@Grand-Thibault and @stephanbcbauer: Can we add the label "Working Model"?

**Reply by ma3u** [reply]
Working Group vs. Expert Group

Expert Group:
- defined process (Application, dedicated persons)
- define stabdards

Working Group:
- Special topic

**Reply by ma3u** [reply]
Resposponsibility of Standards by Committee and Experts. IP Check required by the participating companies required. Review Catena-X Roadmap for standards for the next release: https://github.com/orgs/catenax-eV/projects/15/views/2

**Comment by ma3u** [comment]
** Standards

@jSchuetz88 we need some guidance: 

How is the **application of standardization** from Eclipse Tractus-X in the picture of the [page in Working Model](https://catenax-ev.github.io/docs/next/working-model/ecosystem)? We need a transparent overview of responsible standards per comimittee and responsibility. Also the standard document need the information about the responsible Comitee and Expert expert group.

NSC already have an [overview](https://github.com/catenax-eV/cx-network-service/blob/main/docs/committee/README.md) and we suggest to publish this markdown to the catena-x e.v. website: https://catenax-ev.github.io

**Reply by jSchuetz88** [reply]
Hi @ma3u, the overview will be published soon, but within the standardization repository and not as a part of the working model.

**Reply by jSchuetz88** [reply]
Hi @ma3u,

just for completeness. The list was created a while ago, you can find it here: https://github.com/catenax-eV/product-standardization-prod/blob/main/src/standard-owners.md

Currently it is only internally available. We will discuss whether we are willing to publish the list within the library.

**Comment by ma3u** [comment]
@Grand-Thibault : Who start the process to close a commitee? for DACI matrix.

**Comment by ma3u** [comment]
@Grand-Thibault:

Additional Roles required?

- Catena-X member
- Tractus-X contributors?

**Comment by ma3u** [comment]
[Committee Review](https://github.com/catenax-eV/cx-working-model/blob/NSC_WorkingModelChanges/docs/organizational-structure/catena-x/committee.md) tasks:

- break down [mission](https://catena-x.net/en/benefits) and [vision](https://catena-x.net/en/vision) from Catena-X association
- Define measuable business goals for next 12 months (OKRs?)
- Input to technical and operative status Sync 
- Take responsibility for standards 
  - [Overview Standard Responsibilities](https://bcgcatenax.sharepoint.com/:x:/r/sites/MitgliederbereichMemberArea_Catena-XAutomotiveNetworke.V/_layouts/15/Doc2.aspx?action=edit&sourcedoc=%7B9f834319-f51e-468c-8e69-1da76f8b50c6%7D&wdOrigin=TEAMS-MAGLEV.teamsSdk_ns.rwc&wdExp=TEAMS-TREATMENT&wdhostclicktime=1720772259595&web=1)
  - Break down to expert groups
  - Review [dependencies](https://jschuetz88.github.io/cx-dependencies/#CX-0018) in Huston
- Take responsibility for products and repositories
   - break down responsibility to expert groups
- Take responsibility Tractus-X KITs
   -   break down responsibility to expert groups
- Give a overview in a sounding event every 6 months 
- Guide through the Release Process
   - [PI Planning, Refinement](https://github.com/orgs/catenax-eV/projects/28/views/2) tasks in of C-X expert groups and T-X developer teams
   - Review: Definition of Entry (feature template define the Acceptance Criteria)
   - Tracking of the progress
   - Review: Definition of Done (Quality Gate Issue)

**Comment by ma3u** [comment]
Update Committee page:
https://github.com/catenax-eV/cx-working-model/blob/NSCworkingsession1/docs/organizational-structure/catena-x/committee.md

**Comment by ma3u** [comment]
A [Working Group](https://catenax-ev.github.io/docs/next/working-model/organizational-structure/catena-x/working-groups) should be follow the Expert Group process as light version.

**Comment by ma3u** [comment]
Catena-X Committee and Expert Group organigram missing. @Grand-Thibault, @stephanbcbauer

**Reply by Grand-Thibault** [reply]
The organigram is linked in the description of both. It wont be included in the WM as the C-X homepage should be the single source of truth.

**Comment by ma3u** [comment]
[C-X office](https://catenax-ev.github.io/docs/next/working-model/organizational-structure/catena-x/catena-x-office): Does a release management Expert group is planned?

**Comment by ma3u** [comment]
discussion on demotion of expert group member definition:
+ https://github.com/catenax-eV/catenax-ev.github.io/discussions/129

- [Working Model Expert Group Member](https://catenax-ev.github.io/docs/next/working-model/organizational-structure/catena-x/expert-group#expert-group-member)
